pkg/disk/controllerserver.go

```diff
@@ -103,6 +103,20 @@ const (
 	MinimumDiskSizeInGB = 20
 	// MinimumDiskSizeInBytes ...
 	MinimumDiskSizeInBytes = 21474836480
+	//volumeExpandAutoSnapshotIDKey is the key of volumeExpandAutoSnapshotID on pvc
+	volumeExpandAutoSnapshotIDKey = "volumeExpandAutoSnapshotID"
+	// volumeExpandAutoSnapshotNameKey is the key of volumeExpandAutoSnapshotIName on pvc
+	volumeExpandAutoSnapshotNameKey = "volumeExpandAutoSnapshotName"
```

Comment:
IMO, Snapshot name is not necessary to mark on pvc?

Reply:
controllerServer manages a map to record the created snapshot,  with the names of snapshot as the keys.

Reply:
Sure it's okay to only mark the ID if the volumeExpandAutoSnapshots are not recorded in the controllerServer then the nodeServer will not need to delete it anymore.